.circleci/config.yml

```diff
@@ -5,9 +5,9 @@ common-steps:
     run:
       name: Install system-level and Python dependencies
       command: |
-        apt-get update && apt-get install -y --no-install-recommends make python3 python3-venv python3-setuptools
-        python3 -m venv .venv
-        ./.venv/bin/pip install --require-hashes -r requirements/requirements.txt
+        apt-get update && apt-get install -y --no-install-recommends make python3 python3-pip
+        pip install poetry
+        poetry install --no-ansi
```

Comment:
The `--no-ansi` argument is necessary due to a bug in CircleCI: https://github.com/python-poetry/poetry/issues/7184 (and harmless once that bug is fixed).